salt/utils/http.py

```diff
@@ -132,6 +132,7 @@ def query(url,
           handle=False,
           agent=USERAGENT,
           hide_fields=None,
+          raise_error=True,
```

Comment:
On the contrary, since v4.1.0 it supports this parameter to overwrite [default value which is True](http://www.tornadoweb.org/en/stable/releases/v4.1.0.html?highlight=raise_error#tornado-httpclient), also [documentation](http://www.tornadoweb.org/en/stable/httpclient.html#tornado.httpclient.HTTPClient.fetch) for current stable tornado confirms it.